src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropertyPageSerializationService_Store.vb

```diff
@@ -459,7 +459,6 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
 
                     If TypeOf Member Is PropertyDescriptor Then
                         Dim Prop As PropertyDescriptor = DirectCast(Member, PropertyDescriptor)
-                        Debug.Assert(Prop.PropertyType.IsSerializable)
```

Comment:
IEnumValue is the interface but at runtime, it'll be some type implementing that interface - is that type serializable? More importantly though - do we understand why they were asserting that this type was serializable? Does this code try to serialize the object somewhere?